src/app/containers/HomePage/index.jsx

```diff
@@ -0,0 +1,65 @@
+import React, { Fragment } from 'react';
+import { bool, string, shape } from 'prop-types';
+import nanoid from 'nanoid';
+import HeaderContainer from '../Header';
+import FooterContainer from '../Footer';
+import { ServiceContextProvider } from '../../contexts/ServiceContext';
+import { RequestContextProvider } from '../../contexts/RequestContext';
+import GlobalStyle from '../../lib/globalStyles';
+import nodeLogger from '../../helpers/logger.node';
+import ConsentBanner from '../ConsentBanner';
+
+const logger = nodeLogger(__filename);
+
+/*
+  [1] This handles async data fetching, and a 'loading state', which we should look to handle more intelligently.
+*/
+const HomePageContainer = ({ loading, error, data, bbcOrigin }) => {
```

Comment:
Seeing as these are likely to have very similar `loading`, `error` and `data` handling, could we maybe share some logic?

Reply:
Yup, we should have a dedicated Loading component/container and also too what else we can combine.